terraform/bbq/project_iams.tf

```diff
@@ -36,3 +36,21 @@ resource "google_project_iam_member" "project_bigqueryjobuser_iam" {
   role    = "roles/bigquery.jobUser"
   member  = "serviceAccount:${var.bbq_project}@appspot.gserviceaccount.com"
 }
+
+resource "google_project_iam_member" "project_bigquerydataeditor_iam" {
+  project = "${var.bbq_metadata_project}"
+  role    = "roles/bigquery.dataEditor"
+  member  = "serviceAccount:${var.bbq_project}@appspot.gserviceaccount.com"
+}
```

Comment:
does this roles allow bbq service account to perform export?
in other words, does it the reason why you removed command assigining Datastore Export... role from documentation?

Reply:
Discusses offline :)